### PR TITLE
adding staging.rb,adding secret_key for staging,adding staging in the…

### PR DESCRIPTION
**Pull request by MwMaki**
adding staging.rb,
adding secret_key for staging,
adding staging in the gem group with development and test,
adding database_yml_example

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ gem 'jquery-rails'
 gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
 
-group :development, :test do
+group :development, :test, :staging do
```
Comment (ssharba) [inline Gemfile]: staging will be exactly the same as production environments

File: config/environments/staging.rb [hunk 2]
```diff
@@ -0,0 +1,98 @@
+Rails.application.configure do
```
Comment (ssharba) [inline config/environments/staging.rb]: did you copy this from production? if not please do it.

Comment (MwMaki) [reply config/environments/staging.rb]: yes I did

File: Gemfile (hunk 1 above)
Comment (MwMaki) [reply Gemfile]: ok

**Pull request merged**